hasher-matcher-actioner/hmalib/common/models/bank.py

```diff
@@ -442,7 +460,10 @@ def update_bank(
         if is_active != None:
             bank.is_active = bool(is_active)
 
-        if bank_name or bank_description or (is_active != None):
+        if tags != None:
```

Comment:
blocking? because `tags` defaults to `set()` I think this is always going to be true
same for the conditional on the new 466 line.

Reply:
When received from the UI, we do not get a patch request, so this is not a bug, but I agree with you. This makes things confusing. I'll make the argument optional and default it to None.